server/utils/placementRequests/requestForPlacementSummaryCards.test.ts

```diff
@@ -0,0 +1,165 @@
+import {
+  RequestForPlacementSummaryCards,
+  mapRequestsForPlacementToSummaryCards,
+} from './requestForPlacementSummaryCards'
+import { applicationFactory, requestForPlacementFactory, userFactory } from '../../testutils/factories'
+import paths from '../../paths/apply'
+import { DateFormats } from '../dateUtils'
+import { RequestForPlacementStatusTag } from './statusTag'
+
+describe('RequestForPlacementSummaryCards', () => {
+  const actingUserId = '123'
+  const applicationId = '456'
+
+  describe('if the RfP cannot be directly withdrawn', () => {
+    it('should generate a summary card for the request section', () => {
```

Comment:
would it be more accurate to say that this test is demonstrating that we see both a summary card and an empty list of available actions? e.g.

`it('should generate a summary card for the request section AND include no available actions', () => {`

which makes me think that it might be clearer to describe the behaviour in 4 tests grouped in 2 scenarios:

```
describe('if the RfP cannot be directly withdrawn', () => {
    it('should generate a summary card for the request section'
    it('should not offer any actions'

 describe('if the RfP can be withdrawn', () => {
    it('should generate a summary card for the request section'
    it('should include a withdraw action item'
```